### PR TITLE
Feature/life events

### PR DESCRIPTION
**Pull request by saniemi**
Life events test data and improvements to the prototype matching code.

### CODE REVIEW

File: DataScience/Analytics/linking/addressLinking.py [hunk 1]
```diff
@@ -698,6 +734,11 @@ def _find_likeliest_address(self, addresses_to_be_linked, blocking=1):
         compare.string('LOCALITY', 'Locality', method='jarowinkler', name='locality_dl',
                        missing_value=0.5)
 
+        # add a comparison of the incode - this helps with e.g. life events addresses
+        if self.settings['expandPostcode']:
+            compare.string('postcode_in', 'postcode_in', method='jarowinkler', name='incode_dl',
```
Comment (Mironor) [inline DataScience/Analytics/linking/addressLinking.py]: You are comparing string with the same string?

Comment (saniemi) [inline DataScience/Analytics/linking/addressLinking.py]: The first two positions define which dataframe columns hold the information being compared. In this case the column names are the same, but one refers to the input dataframe while the other to the AddressBase dataframe.

**Comment by iRhysBradbury** [comment]
Remember to squash :)

File: DataScience/Analytics/prototype/lifeEventsAddresses.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+Version
+-------
+
+:version: 0.2
+:date: 14-Dec-2016
+"""
+from Analytics.linking import addressLinking
+import pandas as pd
```
Comment (saniemi) [inline DataScience/Analytics/prototype/lifeEventsAddresses.py]: Didn't know :-)... to my knowledge pd is a common abbreviation for pandas, like np is for numpy, or sp for scipy, so I'll stick to that.

**Pull request merged**